Heron- Class!

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request


![2023 01 19-16 00 15](https://user-images.githubusercontent.com/79304582/213476321-be993cff-b0ac-488b-9c3b-6f33002130b3.png)


![image](https://user-images.githubusercontent.com/79304582/211120744-02f54294-6f8b-4bfd-9904-e6f6df9792d8.png)


This PR adds the Heron-Class Star Dreadnaught (ADMINSPAWN ONLY)!!! It's a gigantic Nanotrasen Vessel, containing a Dropship, a Mechbay, and a full Emergency Response Team!

I created this ship as a kind of "opposing force" to the Twinkleshine, though it would probably heavily differ, in use. 
while it has less soldier capacity, less combat gear, and less medical equipment, it makes up for it with its Marauder mech, Functional"tm" Supermatter engine, and its state-of-the-art, totally radical, dropship. 


the crew consists of:
___
- 1x Captain (Centcom Officer)
- 1x First Officer (hop)
- 1x HOS
- 2x Pilots (for the dropship)
- 1x Security Officer (Internal, actual Security work)
- 6x ERT Officers 
- 1x Mech Pilot
- 1x Engine Technician (Engineers)
- 1x Chief Engineer
- 1x Roboticist
- 2x Medical workers (Paramedic and Medical doctor)
- 1x Atmos tech
- 2x Cargo Crew (quartermaster and cargo tech)
- 1x Cook
- 1x Janitor
____

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
More big ships for admins and players to have fun with is good!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: adds the Heron Class Dreadnaught
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
Update jungle_paradise.dmm

spockye committed:
main files

heron.json
nanotrasen_heron.dmm
captain.dm
cargo_tech.dm
security officer.dm

spockye committed:
misc fixes and job changes

json fixes
map fixes and changes
ship area "Dock" added
pilot changes
officer changes

Sun-Soaked commented:
<img width="789" alt="Screen Shot 2023-01-06 at 7 15 17 PM" src="https://user-images.githubusercontent.com/45698967/211120920-b4a9f985-6ce7-480b-93d4-a5a65b773758.png">
criminal behavior detected. Deploying memetic kill agents

spockye commented:
> <img alt="Screen Shot 2023-01-06 at 7 15 17 PM" width="789" src="https://user-images.githubusercontent.com/45698967/211120920-b4a9f985-6ce7-480b-93d4-a5a65b773758.png">
> 
> criminal behavior detected. Deploying memetic kill agents

i affirm that im going to find out where you live, and mail you large amounts of packages containing air generating devices, and dust

Sun-Soaked commented:
I could be a Dust Mite for all you know. Maybe you'd be feeding me for years. Didn't think about that did you huh???

spockye committed:
fixes lowercase door

adds missing wire

Erikafox commented:
![firefox_7Kgbb4Ookf](https://user-images.githubusercontent.com/94164348/211132572-982c3c39-7595-420a-babf-b8e6d6c59109.png)
Whoever did this is evil


Ping me on the discord for a review tomorrow.

spockye committed:
Update nanotrasen_heron.dmm

spockye committed:
doubled and unconnected pipes

spockye committed:
missing pipe

spockye committed:
attempts to fix pipes

spockye committed:
fixes piping error

100% totally did it this time
swearsies!!

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
poster revolution

removes breakey posters
adds some new posters

Erikafox commented on _maps/configs/heron.json:
```diff
@@ -0,0 +1,68 @@
+{
```
this file needs to be renamed to fit into the schema

Erikafox commented on _maps/configs/heron.json:
```diff
@@ -0,0 +1,68 @@
+		},
+		"ERT Officer":{
+			"outfit": "/datum/outfit/job/security/nanotrasen/ert",
+			"slots": 6
```
I'd divide this down into having at least an engineering and medical ert fellow too.

spockye replied:
it now has one ERT Medical Officer
and one ERT Engineering Officer
both with their respective equipment on spawn

Erikafox commented on code/game/area/ship_areas.dm:
```diff
@@ -306,6 +306,10 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
+/area/ship/security/dock
+	name = "Shuttle Dock"
+	icon_state = "security"
+
```
commenting here as a reminder, no need to do anything right now

goober3 replied:
^^ remove this area! we now have ship/hangar fulfilling the same role.

Erikafox commented on code/modules/jobs/job_types/captain.dm:
```diff
@@ -62,6 +62,13 @@
 	head = /obj/item/clothing/head/caphat/nt
 
+/datum/outfit/job/captain/nt/heron
+	name = "Captain (Nanotrasen)"
```
Ping a lorehead for a better name here. The centcomm clothes tell me they're a bit above a mere Captain.

spockye replied:
changed to Fleet Captain!

Erikafox commented on code/modules/jobs/job_types/cargo_technician.dm:
```diff
@@ -65,6 +65,14 @@
+	suit = /obj/item/clothing/suit/jacket/leather/duster
+	glasses = /obj/item/clothing/glasses/hud/spacecop
+	accessory = /obj/item/clothing/accessory/holster
+
```
I'm starting to think we need to just wire up a pilot job.

Erikafox commented on _maps/configs/heron.json:
```diff
@@ -0,0 +1,68 @@
+		"Janitor": 1
+
+	},
+	"enabled": true
```
set false for adminspawn.

Erikafox reviewed (changes requested):
The ship doesn't always spawn. 
![dreamseeker_t6Ctu3boja](https://user-images.githubusercontent.com/94164348/212397271-b83f0fe3-8f54-4590-aab9-bcc4294a2f20.png)
needs a bypass so people don't feel obligated to pull gasses out, I'd do something like...
![StrongDMM_ncccgB5ozX](https://user-images.githubusercontent.com/94164348/212397465-e32e458d-fefc-4d4e-ad96-509f1fe03bb9.png)
![StrongDMM_1oc0AgM0Ed](https://user-images.githubusercontent.com/94164348/212397530-3d620df6-cb49-4ef9-973a-86eb785a4856.png)
Take the shutters out of the walls
![image](https://user-images.githubusercontent.com/94164348/212397641-fc7de8c5-b5e3-4a00-98d7-71e674695cfb.png)
swap the upper airlock for a glass one, put an air alarm in there, like the screencap.
![image](https://user-images.githubusercontent.com/94164348/212397930-a76e9626-cb6f-4d05-a665-ed34ecda6bb7.png)
the holopad is cute, but any callers will be offset from it.
![StrongDMM_cwOMY1YhMJ](https://user-images.githubusercontent.com/94164348/212398022-d01e2625-bb30-47ae-a701-d80a2fb2262c.png)
toss a button on the inside of the cargo area, so you can't just trap people in there
![dreamseeker_5QdB05PHD3](https://user-images.githubusercontent.com/94164348/212398144-f730dabf-61a4-4787-9226-5e79125cafab.png)
minor decal melding, if you can figure out a way to make it have less of that, that'd be bitchin
![image](https://user-images.githubusercontent.com/94164348/212398278-3fde7266-2080-4cf8-b3e6-9b8c2f06de11.png)
Give them some suit variety, preferably to match the variety in roles (as mentioned in file review).
![dreamseeker_bbIM4nukDg](https://user-images.githubusercontent.com/94164348/212398639-62fa223c-f48b-443e-963b-464854d9bcb0.png)
Decal merging
![StrongDMM_pPcmAw8Im9](https://user-images.githubusercontent.com/94164348/212398822-bf7211cc-2bdb-4ca6-b157-478c68e46b0c.png)
Give the hardsuit it's own name, clean the captain's bathroom.
![dreamseeker_iKe6DYCQQH](https://user-images.githubusercontent.com/94164348/212398913-8fe78700-8926-47ca-9946-390ee05315dc.png)
You should make this a sprite instead of pixelshifting 3 together.
![dreamseeker_HXgU3O80NQ](https://user-images.githubusercontent.com/94164348/212398990-2d16ddba-d930-4555-a26f-e514fa3a7fb0.png)
You can just pull the medals off the wall, I'd recommend using a /structure for that instead of the items.
![dreamseeker_BhfeS2G2fg](https://user-images.githubusercontent.com/94164348/212399050-4f8b3a94-29ec-46c7-ba3d-65fc3a67991f.png)
James (59)
![StrongDMM_bGwmbM2Q6L](https://user-images.githubusercontent.com/94164348/212401459-2aa988e0-8885-4b1f-b423-0f0b15cdc833.png)
I would turn this area into a proper like.. Checkpoint?
![StrongDMM_2T3xlzUYH9](https://user-images.githubusercontent.com/94164348/212401590-c6913db0-39d6-4870-b264-a43275f407dc.png)
For a big ship that I'm sure will be seeing a decent bit of downtime, there's just not enough of a crew area. I'd push it out into the hallway and giving them a bit more of a lounge - chill space.
![StrongDMM_o4WEgmCOPs](https://user-images.githubusercontent.com/94164348/212403423-6015d0d2-df31-4905-b9b0-624c7ee03ed1.png)
You're gonna need another vent here
![StrongDMM_rgI5lzXZ3o](https://user-images.githubusercontent.com/94164348/212403775-1e82fa08-7134-42d4-a19c-6c6dbc37ab51.png)
Add some markings for the orange pipes here.
![StrongDMM_MlDYSIdeGv](https://user-images.githubusercontent.com/94164348/212405873-7aaa4217-74f1-4a9c-a0ad-1fdaa3b8ebd1.png)
Same deal with the cafe, it could stand to eat into the hallway a bit
![StrongDMM_k0TGMBV1ag](https://user-images.githubusercontent.com/94164348/212406019-6f4429d4-58b0-4dd5-805e-bb021564a37c.png)
This airlock needs another vent too
![StrongDMM_DdCOQ0Zgha](https://user-images.githubusercontent.com/94164348/212406140-a0768643-6784-4fdd-bf36-f1092dc4ed46.png)
Give the AI a blast door or shutters tbh ngl
![StrongDMM_4sKaa4EDZF](https://user-images.githubusercontent.com/94164348/212406226-6ad1ec3c-8978-4465-8f18-269577e5caaf.png)
Box reflector in the middle, they need it if all emitters are gonna be firing.

I'm sure there'll be more but I'm feeling map fatigue, so I'll give another go over once you do all this up

spockye committed:
misc fixes and changes

json changes(
captain -> Fleet Captain
ERT officer 6<4
ERT Engineering Officer
ERT Medical Officer
)
Map changes(
extra table outside canteen
wall shutter gone
plasma subship fuelline marker
hardsuit changes (captain hardsuit/ERT member hardsuits/Medic Hardsuit)
ai shutter
roboticist spare clothing
snowflakes wall-medals
snowflakes decal layers to prevent improper overlaying
gives the poor janitor a souvernier of his glory days.

spockye committed:
majorly updates the Crew area

pretty big changes to the layout

spockye committed:
misc fixes

SM fixes

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
fixes Piping (thanks Karl)

Done by "Blessed Karl#0184"

Erikafox commented:
![image](https://user-images.githubusercontent.com/94164348/213482295-f3846cbf-c618-481e-96aa-d3bd98bf66fb.png)
If you really wanted the medals to look fancy, you could layer a notice board under them.

Latentish commented:
I need to be able to see your area layout.

Latentish commented:
Does that droppod actually detach?

spockye commented:
> Does that droppod actually detach?

not at the moment. waiting for subshuttles to get merged

spockye committed:
misc changes

adds 3 assistants
replaces a ENGI suit with a CE suit
armoury +2 barriernade / 1x sabre smg
forensics scanner
changed brig doors to glass
added missing cameras
misc decal changes
secborg modules
another posi for the ai core
2 more pickaxes and a basic drill
crate with weak spess suits
2x synthflesh sprays to med

spockye committed:
subshuttles the subshuttle

Erikafox commented:
drafted like all other subship reliant prs

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
fixes conflict

goober3 commented:
north side cargo windoor is access locked!

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

Erikafox reviewed (dismissed):
Needs an exterior subshuttle port

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
misc fixes and changes

adds access locks to engineering
replaces softengi suit with HARD
adds a deployable machine gun
removes sec vendor
replaces holofloors
replaces HOS locker
changes some access restrictions
adds subshuttle dock

giv hug :(

spockye commented:
ready for re-review

spockye committed:
internal dock possible fix

dont blame it on the sunshine!
dont blame it on the moonlight~
dont blame it on the good times :surfer:
blame it on the boogie :dancer:

spockye committed:
misc updates

Addust commented:
STINKY nontitanium window, replace or i will send you to god with the hyena horde
<img width="215" alt="image" src="https://github.com/shiptest-ss13/Shiptest/assets/80979251/347c3c78-61e4-4e80-aca3-ac2bccc98c80">

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
misc changes

attempts to fix subshuttle

replaces sec hardsuits from LP to real

misc fixes

spockye committed:
oopsies

additional subdock

spockye committed:
FIXES the subshuttle

actually does it this time

i pinky promise
:)

spockye committed:
access locks the AI core

spockye committed:
fixes hos locker

oopsies

spockye committed:
misc changes

adds some decals

replaces a inducer with another RPD

adds a missing light

spockye committed:
switches some windoors around

spockye committed:
moves the buttons on the subshuttle

spockye committed:
misc additions and changes

makes a "riged_steel" floor type
(uses a unused sprite)

uses this new tile at the mechbay

adds some misc decals around the ship

spockye committed:
misc fixes

decal fixes

adds a holopad to the subshuttle

spockye committed:
changes and fixes

adds a CE role
removes a engineer role
fixes paramedic being uncapitalized
adds some produce

spockye committed:
slightly changes the kitchen

spockye committed:
decal changes

spockye committed:
removes the bounty console

it doesnt work
replaces it with a frame

spockye committed:
misc small fixes

chairs are no longer dark
ssu's no longer eat the intercoms
uses real subtypes
adds a sink and belt in jannies

spockye commented:
stop failing ;(

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
more jannie gear

gives the janitor
a locker with clothes
replaces the belt with the full var
mop/broom/bucket
rag

spockye committed:
Merge branch 'Nanotrasen_capital_1' of https://github.com/spockye/Shiptest into Nanotrasen_capital_1

spockye committed:
adds a proper comms setup.

moves some stuff around to fit it

spockye committed:
removes all-in-one

oopsies

spockye committed:
oops

atmos tiles
oopsie woopsie

spockye committed:
BAD iconstate edit removal

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

Signed-off-by: spockye <79304582+spockye@users.noreply.github.com>

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

Signed-off-by: spockye <79304582+spockye@users.noreply.github.com>

spockye committed:
Merge branch 'Nanotrasen_capital_1' of https://github.com/spockye/Shiptest into Nanotrasen_capital_1

spockye committed:
Merge branch 'Nanotrasen_capital_1' of https://github.com/spockye/Shiptest into Nanotrasen_capital_1

spockye committed:
conflict fix

Addust commented:
voidcrew beat you to it man

Erikafox committed:
Update nanotrasen_heron.dmm

Erikafox committed:
Merge branch 'Nanotrasen_capital_1' of https://github.com/spockye/Shiptest into pr/1670

spockye committed:
atmos fixes (no recycle loop for now)

spockye committed:
adds a camera to tcomms

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

Merlin1230 commented:
weird error I found loading up this branch for shits and giggles, this ship upon loading immediately deletes itself
![image](https://github.com/shiptest-ss13/Shiptest/assets/76177064/3709d498-2d86-4e4a-832f-0a476367d8b4)
![image](https://github.com/shiptest-ss13/Shiptest/assets/76177064/6a773fb6-dde7-4acc-a94c-8b202486c6c3)
no clue why, doesnt seem to happen with other ships, but thought I would bring it up

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

Signed-off-by: spockye <79304582+spockye@users.noreply.github.com>

spockye commented:
yea, happens on my side aswell... i'll see if i can figure out why

spockye committed:
fixes some illegal floors

spockye committed:
Merge branch 'Nanotrasen_capital_1' of https://github.com/spockye/Shiptest into Nanotrasen_capital_1

spockye committed:
misc decal fixes

spockye committed:
misc atmos fixes

forgor to re-add everything to atmos

woopsies

spockye committed:
fixes illegal floors on the subshuttle

spockye committed:
gives AI core robotics access

spockye committed:
removes a bee plushieeee

1, sure
but two!???
am i out of my mind!?!?!?!?

spockye committed:
moves a poster

(it was on top of a mirror)

spockye commented:
seems like it's fixed, unable to reproduce it

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye commented:
Drafted until im back

Imaginos16 commented:
they never came back.......................

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye commented:
Erika TRAPPED ME for over a week. I'll get on fixing this now

spockye committed:
Update nanotrasen_heron.dmm

spockye committed:
Update nanotrasen_heron.dmm

spockye committed:
how about you script deez

spockye committed:
fixes wallmounts

And so I cry sometimes

when I'm lying in bed
Just to get it all out
what's in my head
And I...
I'm feelin',
a little peculiar

And so I wake in the morning
and I step outside
And I take a deep breath
and I get real high,
and I
scream from the top of my lungs
What's going on?

And I say,

hey ye yaae yae yaa,
hey ye yaae yaae
I said hey!
What's going on?
And I say,
hey ye yaae yae yaa,
hey ye yaae yaae
I said hey!
What's going ooooon?

[Spoken] AND HE TRIES

oh my god do I try
I try all the time,
in this institution

[Spoken] AND HE PRAYS!
Oh my god do I pray!
I pray every single day
[Spoken] MNYAAAAH!!
For a revolution!
WHOOOO!

And I say,

hey ye yaae yae yaa,
hey ye yaae yaae
I said hey!
What's going on?
And I say,
hey ye yaae yae yaa,
hey ye yaae yaae
I said hey!
Learn how to hide
your feelings
hey ye yaae yae yaa,
heeey ye yaae,
yaae yaae yaaeeh
I said hey!
What's going on?

YYEEEEEEEAAAAAAHHHH

spockye committed:
more fixes

spockye committed:
fixes the subshuttle

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
misc fixes

robotics dirs
cameras floating
comms atmos

spockye committed:
more misc fixes

spockye committed:
more fixes

spockye committed:
fixed air injectors starting off

spockye committed:
Update nanotrasen_heron.dmm

spockye committed:
Misc changes

repath of Pilot to be a HOP subtype

removed two ert officer slots + removed paramedic

spockye committed:
more job cuttings

removes 1 assistant slot
removes 1 pilot slot

spockye committed:
re-adds ERT slots

spockye committed:
fixes and adds more extinguishers

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
misc fixes

fixes:
fax machine type
Misc outfits

spockye committed:
fixes robotics role

goober3 reviewed (changes requested):
![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/413aae5c-4483-4f99-abd9-ffb665eb2a56)
![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/7de5e660-6a1d-44df-8ed8-ec280cba30ed)
intercoms need pixel shifting

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/709ce96d-7101-475b-8d28-83aa9d1d37d3)
these walls need to use the new wall sprites or be replaced.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/493e37d7-8f48-45c5-870f-2dbdb3ef8ade)
unrotated shutters

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/966b75c0-1a11-4a8c-911b-828e98cfff87)
poster light

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/169632b8-659c-4cbf-9e55-68cb556ffdb9)
unconnected wire in the emitter room.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/06368de5-707e-49f9-aa43-b537375f7809)
none of these emitters will reflect into the SM like this! please replace the reflectors with reflector boxes, or remove the outer emitters.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/60783f87-af6d-42ad-babb-036c4778a27b)
this radiation closets will get shoved out of place quite easily!! consider securing them, somehow.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/45c76214-3ddd-4a33-9d40-bb8af539f4fc)
i'd rather these high security doors be replaced with something else, for as long as they lack anywall sprites.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/0bad5db5-aea6-449f-99f7-08073a347c47)
the inner windoor on the desk here should be access restricted.

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/68fdae3e-7b33-4899-ba16-371bc0c8bf71)
this looks nice but please don't pixelshift holopads!! 

![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/117adfc7-b1e1-479b-a552-dd178cced432)
this room needs a vent & scrubber.

this ship still needs tags & a description!
every area needs a light switch. (this applies to the shuttle, too)
please replace the exterior windows with non-spawner versions.

spockye committed:
Merge branch 'master' into Nanotrasen_capital_1

spockye committed:
Bunch of review related fixes

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
missing lightswitches

also adds a picture frame

meemofcourse commented on _maps/shuttles/shiptest/nanotrasen_heron.dmm:
in the subshuttle dock, you are using area/hangar, and this causes an immediate sgt

spockye replied:
Ill get that fixed in a day or two

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
fixes misc stuffs

fixes hangar area
adds missing light switches

spockye committed:
TEMP BRIDGE REMAKE

testing out a new bridge

making it different later on

spockye committed:
misc bridge oopsies

spockye committed:
more misc bridge oopsies

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
fixed missing helm

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
misc updates and fixes

spockye committed:
pilot shemags

spockye committed:
Merge remote-tracking branch 'upstream/master' into Nanotrasen_capital_1

spockye committed:
adds Description (thanks Meem :)))) )

also adds some power cells around

goober3 committed:
Merge branch 'master' into Nanotrasen_capital_1

goober3 commented:
good! i'd nitpick this more but this ship been in the oven for long enough.

Doubleumc commented:
Emitter reflectors start oriented such that they will reflect back at the emitter, destroying it.

No SM monitor console.
